.github/workflows/release.yml

```
@@ -1,70 +1,25 @@
-on: release
+on:
+  release:
+    types: [published]
+    
 name: Build Release
 jobs:
-  release-linux-amd64:
-    name: release linux/amd64
+  amd64-releases:
```

```suggestion
  release-bins:
```

since it does more than amd64